x-pack/plugins/security_solution/public/detections/components/alerts_kpis/common/hooks.test.tsx

```diff
@@ -144,7 +144,7 @@ describe('hooks', () => {
       const { result, unmount } = renderHook(() => useStackByFields(useLensCompatibleFields), {
         wrapper,
       });
-      const aggregateableFields = result.current;
+      const aggregateableFields = result.current();
       unmount();
       expect(aggregateableFields?.find((field) => field.label === '@timestamp')).toBeUndefined();
       expect(aggregateableFields?.find((field) => field.label === '_id')).toBeUndefined();
```

Comment:
```suggestion
      expect(aggregateableFields!.find((field) => field.label === '@timestamp')).toBeUndefined();
      expect(aggregateableFields!.find((field) => field.label === '_id')).toBeUndefined();
```

Not related to your changed but I noticed that these tests will pass in case `aggregateableFields` is `undefined`. One way to fix that is to swap that `?` for a `!` to make them fail at runtime in case the hook is broken. Wdyt?

Reply:
The same could be done in `returns only aggregateable fields`.

Reply:
ok!